app/javascript/modules/listings/SharedHelpers.tsx

```diff
@@ -70,12 +70,16 @@ export const getImageCardProps = (listing: RailsListing, hasFiltersSet?: boolean
       ? listing.Listing_Images[0].displayImageURL
       : listing?.imageURL ?? fallbackImg
 
+  const imageDescription =
+    listing?.Listing_Images?.length > 0
+      ? listing.Listing_Images[0].Image_Description
+      : `${listing.Building_Name} Building`
```

Comment:
I know that this was like this before, but I'm wondering if this text should be translated?

Reply:
Yeah we could dynamic translate the salesforce alt text and provide translations for the fallback.

Reply:
Adding the `useTranslate()` hook call to a parent component ( I placed it in `RentDirectory` ) seems to work. But not sure if this is the best way? I can create a ticket. In Angular it looks like we are translating alt text successfully.